src/sentry/templates/sentry/emails/sentry-app-publish-confirmation.html

```diff
@@ -0,0 +1,33 @@
+
+{% extends "sentry/emails/base.html" %}
+{% load i18n %}
+{% load sentry_avatars%}
+{% load sentry_helpers %}
+
+{% block main %}
+<p>
+  Thank you for your submission!
+  This email confirms that user {{ actor.email }} has submitted {{ sentry_app.slug }} for {{ organization.slug }}.
+</p>
+<table>
+  <div style="padding-bottom: 8px;"> Here are your responses to the submission form: <div>
+	{% for question in questionnaire %}
+  <tr>
+    <td>
+      <div style="padding-bottom: 8px;"><strong>{{ question.question }}</strong></div>
+      <div style="padding-bottom: 8px;"> > {{ question.answer }}</div>
```

Comment:
can we add a bit more styling to differentiate this as an answer? like a border on the div?

Reply:
like one of these (if so which one)?
<img width="869" alt="image" src="https://github.com/user-attachments/assets/6fa15b89-6859-466a-831c-197d719a28fc" />

Reply:
can you find the styling for this Exception block? perhaps it would look better like that

<img width="704" alt="Screenshot 2025-02-12 at 12 04 21" src="https://github.com/user-attachments/assets/0cead759-8605-4828-bfad-aba728233765" />